lms/djangoapps/teams/models.py

```diff
@@ -15,8 +15,8 @@ class CourseTeam(models.Model):
     """This model represents team related info."""
 
     team_id = models.CharField(max_length=255, unique=True)
-    name = models.CharField(max_length=255)
-    is_active = models.BooleanField(default=True)
+    name = models.CharField(max_length=255, db_index=True)
+    is_active = models.BooleanField(default=True, db_index=True)
```

Comment:
I thought that indexing on a boolean field didn't do much of anything. It would be interesting to run your test with and without the Boolean index.